pkg/apis/quota/group.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 The KubeSphere Authors.
```

Comment:
Copyright should be consistent in the commits.
```suggestion
Copyright 2021 The KubeSphere Authors.
```

Reply:
fixed